deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/config.json

```diff
@@ -0,0 +1,64 @@
+{
+  "Version": 1,
+  "Identity": {
+    "PeerID": "",
+    "PrivKey": ""
+  },
+  "Assignment": {
+    "FilterIPs": false,
+    "IndexerPool": [
+      {
+        "AdminURL": "http://ber.dev.cid.contact:3002"
+        "IngestURL" "http://ber.dev.cid.contact:3001"
+      },
+      {
+        "AdminURL": "http://cali.dev.cid.contact:3002"
+        "IngestURL" "http://cali.dev.cid.contact:3001"
+      }
+    ],
+    "Policy": {
+      "Allow": true,
+      "Except": null
+    },
+    "PubSubTopic": "/indexer/ingest/mainnet",
+    "Replication": 0
```

Comment:
Can you please remind me what replication `0` means?

Reply:
Replication 0 (default), means to assign publisher to all indexers in pool.